README.md

```diff
@@ -48,6 +48,13 @@ The simplest way to start the process editor is by download a [Axon Ivy Designer
 
 ---
 
+## Changing the protocol
+
+1. Change the schema in the core
+2. Run build core_json-schema
+3. Copy the URI of the schema protocol and put it in `packages/inscription-protocol/schemaCodegen.cjs`
+4. Run `npm run generate` in directory `packages/inscription-protocol`
```

Comment:
`npm run protocol generate` in root should also work

Reply:
and it is also possible to add an arg to this generate cmd with a URL to a valid new schema json